test/built-ins/GeneratorPrototype/return/try-finally-set-property-within-try.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022 Bo Pang. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 25.3.1.3
+description: >
+    When a generator is paused within a `try` block of a `try..finally`
+    statement, `return` should interrupt control flow as if a `return`
+    statement had appeared at that location in the function body.
+    And the `finally` block runs first.
+features: [generators]
+---*/
+
+
+var obj={foo:undefined};
+function* g() {
+  try {
+    obj.foo=yield;
+  } finally {
+    return 1;
+  }
+}
+var iter = g();
+var result;
+
+iter.next();
+let ret=iter.return(45);
```

Comment:
Note the linting failures: https://app.circleci.com/pipelines/github/tc39/test262/4541/workflows/a2e87bc5-695b-4e54-b68e-2ebd6ca03653/jobs/42470#shared-improved-time-tooltips
```suggestion
result = iter.return(45);
```